backend/src/api/instances/[id]/get.handler.ts

```diff
@@ -1,14 +1,26 @@
 import { RequestHandler } from 'opine/mod.ts';
+import { getJimmiBy } from '../../../service/Jimmi.service.ts';
 
 /**
  * @swagger
  * /instances/{id}:
  *   get:
  *     description: Get detailed information about an instance
+ *     parameters:
+ *       - in: path
+ *         name: id
+ *         type: string
+ *         required: true
+ *         description: UUID of Jimmi instance
  *     responses:
  *       200:
- *         description: hello world
+ *         description: Jimmi instance
+ *       404:
+ *         description: No instance found under the given id
  */
-export const getHandler: RequestHandler = (_req, res, _next) => {
-  res.send('I am Jimmi!');
+export const getHandler: RequestHandler = (req, res, _next) => {
+  const id = req.params.id;
```

Comment:
```suggestion
  const { id } = req.params;
```
Should we use object destructuring? Mabye we should enforce/discourage the usage via linting rules.